misc/jobserver_pool.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python3
+# Copyright 2024 Google Inc. All Rights Reserved.
```

Comment:
```suggestion
# Copyright 2024 Google Inc. Some Rights Reserved.
```

The right to use the code for personal use is not reserved in any way.

Reply:
Interesting. I am curious to know why you would want to make this change.

I had to look it up just to be sure.

For the record, "All Rights Reserved" has a very precise definition in international copyright law. And due to changes in international treaties, these words are notwcompletely optional for code written after 2023. Even Google's internal open-source licensing rules recommend omitting them from new code, but some projects can still continue to include it if they want. In this case, I just copy-pasted it from another header, but I would gladly remove it though. @jhasse, what do you think?

On the other hand, "Some Rights Reserved" seems to be something that Creative Commons came up with, but isn't used elsewhere. It doesn't have any value from a legal standpoint, all the rights given to developers, for personal use or not, are listed in the Apache 2.0 license, which is the only source of truth here. And given that there are myriads of CC licenses, many of them are banned by my employer, I'd rather not introduce something that would confuse anyone. Wdyt?

Reply:
It's the same header that was used in every other source file, let's keep it consistent. Also not in scope of this PR.